frontend/communication.tsx

```diff
@@ -2,15 +2,9 @@ import React from 'react'
 
 declare const process: {env: {NODE_ENV: string}}
 
-export const backend_url =
-  process.env.NODE_ENV === 'development'
-  ? 'http://localhost:4200'
-  : '';
+export const backend_url = window.location.href.slice(0, -window.location.pathname.length);
```

Comment:
behöver man sätta backend_url alls här? om vi bara kör samma så går det ju automagiskt till rätt host? eller finns det nån säkerhetsanledning att ha absoluta url'er? isåfall kan du använda `location.origin` direkt för att få url'en iaf!

Reply:
`location.origin` är nog bättre, ja. Men nä, det borde nog inte behövas och las nog mest in för att det fanns.

Jag ser ingen anledning att man skulle behöva ha den absolut.